source/includes/fact-atlas-roles.rst

```diff
@@ -1,15 +1,17 @@
-The user specified in the connection string must have the
+The user specified in the connection string must have, at a minimum, the
 :atlasrole:`atlasAdmin` role.
 
-To use ``mongosync`` in the :ref:`reverse direction <c2c-api-reverse>`,
-you must `create a custom role
-</atlas/reference/api/custom-roles-create-a-role/>`__ that grants the
-following ActionTypes:
+.. note:: 
 
-- setUserWriteBlockMode
-- bypassWriteBlockingMode
-
-The ``setUserWriteBlockMode`` and ``bypassWriteBlockingMode``
-ActionTypes are available starting in MongoDB 6.0. To create the custom
-roles, all clusters in a project must be on MongoDB 6.0 or higher.
+   To use ``mongosync`` in the :ref:`reverse direction <c2c-api-reverse>`,
+   you must `create a custom role
```

Comment:
[link sugggestion]

Consider using a ref for this link rather than than the inline link style.

Reply:
Fixed